WordPressAuthenticatorTests/Credentials/CredentialsTests.swift

```diff
@@ -0,0 +1,127 @@
+import XCTest
+@testable import WordPressAuthenticator
+
+class CredentialsTests: XCTestCase {
+    
+    let token = "arstdhneio123456789qwfpgjluy"
+    let siteURL = "https://example.com"
+    let username = "user123"
+    let password = "arstdhneio"
+    let xmlrpc = "https://example.com/xmlrpc.php"
+    
+    
+    func testWordpressComCredentialsInit() {
+        let wpcomCredentials = WordPressComCredentials(authToken: token,
+                                                       isJetpackLogin: false,
+                                                       multifactor: false,
+                                                       siteURL: siteURL)
+        
+        XCTAssertEqual(wpcomCredentials.authToken, token)
+        XCTAssertEqual(wpcomCredentials.isJetpackLogin, false)
+        XCTAssertEqual(wpcomCredentials.multifactor, false)
+        XCTAssertEqual(wpcomCredentials.siteURL, siteURL)
+    }
+    
+    func testWordPressComCredentialsSiteURLReturnsDefaultValue() {
+        let wpcomCredentials = WordPressComCredentials(authToken: token,
+                                                       isJetpackLogin: false,
+                                                       multifactor: false,
+                                                       siteURL: "")
+        
+        let expected = "https://wordpress.com"
+    
+        XCTAssertEqual(wpcomCredentials.siteURL, expected)
+    }
+    
+    func testWordPressComCredentialsEquatable() {
+        let lhs = WordPressComCredentials(authToken: token,
+                                          isJetpackLogin: false,
+                                          multifactor: false,
+                                          siteURL: siteURL)
+        
+        let rhs = WordPressComCredentials(authToken: token,
+                                          isJetpackLogin: false,
+                                          multifactor: false,
+                                          siteURL: siteURL)
+        
+        XCTAssertTrue(lhs == rhs)
+    }
+    
+    func testWordPressComCredentialsNotEquatable() {
```

Comment:
Like any other naming problem in software development, tests names are hard to assign, but using an appropriate name can go a long way in helping future developers who read the code.

Reading "test `WordPressComCredentials` not `Equatable`" makes me think that the type is not `Equatable`. That per se is something that looking at the type signature would already tell me. Also, the compiler would throw an error if I tried to `==` to instances of a non `Equatable` type.

What this test is doing is proving that two instances with a different `siteURL` value are not equal. How could we rename the test to make it clear?